sonar-python-plugin/src/test/resources/org/sonar/plugins/python/ipynb/file1.ipynb

```diff
@@ -1,3 +1,25 @@
+#SONAR_PYTHON_NOTEBOOK_CELL_DELIMITER
```

Comment:
I think it's a very valid change, however this is a unit test.
ITs are the integration tests located under `its`. Probably we can rename the commit accordingly (and possibly create a new ticket, or update this one - anyway I think improvements on tests in general are very valid).

Reply:
done